bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>Nit, the logo seems too big. Should be able to scale it in notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>It kinda does, it scales with the size of your screen basically. as far as I know Markdown doesn't really support scaling, I could take a smaller image though I guess. </p>